gcp/config.ts

```diff
@@ -9,54 +9,96 @@ function getBooleanOrDefault(key: string, def: boolean): boolean {
   return ret == undefined ? def : ret;
 }
 
-// license-file-path has to be set using cli
-// `pulumi config set akka-cloud-platform-gcp-deploy:license-file-path <value>`
-export const licenseFile = config.require("license-file-path");
-
-export const LightbendNamespace = "lightbend";
-
-export const operatorNamespace = config.get<string>("akka.operator.namespace") || LightbendNamespace;
-export const installTelemetryServices = getBooleanOrDefault("akka.operator.installTelemetryBackends", true);
-
-export const akkaOperatorChartOpts: ChartOpts = {
-  chart: "akka-operator",
-  version: config.get<string>("akka.operator.version") || "1.1.21",
-  fetchOpts: {
-    repo: "https://lightbend.github.io/akka-operator-helm/",
-  },
-};
-
-export function gkeNodePoolArgs(
-  clusterName: pulumi.Output<string>,
-  zone: string | undefined,
-): gcp.container.NodePoolArgs {
-  return {
-    cluster: clusterName,
-    location: zone,
-    initialNodeCount: config.getNumber("gke.nodePool.initialNodeCount") || 3,
-    autoscaling: {
-      maxNodeCount: config.getNumber("gke.nodePool.autoscaling.maxNodeCount") || 7,
-      minNodeCount: config.getNumber("gke.nodePool.autoscaling.minNodeCount") || 1,
-    },
-    nodeConfig: {
-      preemptible: true,
-      machineType: config.get<string>("gke.nodePool.nodeConfig.machineType") || "n1-standard-4",
-      oauthScopes: ["https://www.googleapis.com/auth/cloud-platform"],
+class Namespaces {
+  static readonly LightbendNamespace: string = "lightbend";
+  static readonly AwsOTelCollectorNamespace: string = "aws-otel-collector";
+}
+
+class Versions {
```

Comment:
It should be nice, in the future to share configurations that are not provider-specific across providers

Reply:
Yes! Opened https://github.com/lightbend/akka-cloud-platform-deploy/issues/18.